.gitmodules

```diff
@@ -13,3 +13,6 @@
 [submodule "grpc/server/proto"]
 	path = grpc/server/proto
 	url = https://github.com/dronecore/DroneCore-Proto.git
+[submodule "plugins/mission/libs/json11"]
```

Comment:
Nice to have it inside the plugin :-).

How about putting it in `third_party` instead of `libs`? I started using `third_party` because that's what they use in gRPC and many of its dependencies (maybe that's a Google convention, Idk), and I find it clearer.

Reply:
>Nice to have it inside the plugin :-)

Yes, I placed it inside mission as it is dependency of the plugin alone.
>How about putting it in third_party instead of libs? I started using third_party because that's what they use in gRPC and many of its dependencies (maybe that's a Google convention, Idk), and I find it clearer.

I see. Yeah AFAIK we have a choice: `external` or `third_party`. I'm not particular about it, but I see that `external` is popular. What do you think ?

Reply:
I don't have a strong opinion on this but we should be consistent if possible. If it changes, then let me know because this is used in the docs.

Reply:
I like `third_party`, firstly because there is no possible confusion for me, and also because that's the one I see in grpc, protobuf, boringssl, that are our dependencies already.